Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.h

```diff
@@ -184,8 +184,8 @@ class ITK_TEMPLATE_EXPORT DisplacementFieldTransform : public Transform<TParamet
   virtual void
   SetDisplacementField(DisplacementFieldType * field);
   virtual void
-               SetDisplacementField(VectorImageDisplacementFieldType * field);
-  virtual void SetDisplacementField(std::nullptr_t) = delete;
+       SetDisplacementField(VectorImageDisplacementFieldType * field);
```

Comment:
Why is there this much indentation here? It should be only 2 spaces. Does `clang-format` produce this?

Reply:
Yes, this is the output from running clang-format on the document in Visual Studio. I am not sure why the spacing is different.

Reply:
But since linter on GitHub CI passes, I assume that version would have the same problem.

Reply:
Visual Studio picks up the project clang-format file by default (I typically get a pop-up window for this when I run clang-format for the first time after reopening a project).

Reply:
I was not talking about `.clang-format` style specification file, but about executable `clang-format.exe` which Visual Studio invokes for formatting.